content/docs/dataform-cli/index.md

```diff
@@ -92,6 +92,8 @@ dataform init-creds redshift
 dataform init-creds snowflake
 - or -
 dataform init-creds sqldatawarehouse
+- or, if you've cloned a pre-existing project -
```

Comment:
i think this is in the wrong place - should be next to `dataform init` examples, not `dataform init-creds`, underneath "Create a new project"